docs/quick_start.md

```diff
@@ -0,0 +1,121 @@
+# Quickstart Guide: UiPath LangChain Agents
+
+## Table of Contents
+1. [Introduction](#introduction)
+2. [Prerequisites](#prerequisites)
+3. [Step-by-Step Guide](#step-by-step-guide)
+   1. [Install UiPath LangChain Package](#1-install-uipath-langchain-package)
+   2. [Create Your First UiPath Agent](#2-create-your-first-uipath-agent)
+   3. [Set Up Environment Variables](#3-set-up-environment-variables)
+   4. [Run the Agent Locally](#4-run-the-agent-locally)
+   5. [Pack and Publish the agent on UiPath Cloud Platform](#5-pack-and-publish-the-agent-on-uipath-cloud-platform)
+   6. [Invoke the Agent on UiPath Cloud Platform](#6-invoke-the-agent-on-uipath-platform)
+4. [Next steps](#next-steps)
+
+## Introduction
+
+This guide walks you through the process of setting up, creating, publishing, and running your first UiPath-LangChain Agent.
+
+## Prerequisites
+
+Ensure you have the following installed before proceeding:
+
+- **Python 3.10** or later
+- **pip** command line tool
+
+## Step-by-Step Guide
+
+### 1. Install UiPath LangChain Package
+
+Begin by installing the UiPath LangChain package:
+
+```bash
+pip install uipath[langchain]
+```
+
+### 2. Create Your First UiPath Agent
+
+Generate your first UiPath LangChain agent using the following command:
```

Comment:
`Generate your first UiPath LangChain agent` inside a langchain project that has langgraph.json